assets/src/blocks/Accordion/AccordionEditorScript.js

```diff
@@ -37,9 +37,9 @@ registerBlockType(BLOCK_NAME, {
   icon: 'menu',
   category: 'planet4-blocks',
   keywords: [
-    __('accordion'),
-    __('faq'),
-    __('collapsible')
+    'accordion',
```

Comment:
These don't need translations? 🤔

Reply:
As discussed in sprint meeting, we are excluding the block keywords from translation.